modules/quality/include/opencv2/quality/qualitymae.hpp

```diff
@@ -0,0 +1,78 @@
+#ifndef OPENCV_QUALITY_MAE_HPP
+#define OPENCV_QUALITY_MAE_HPP
+
+#include "qualitybase.hpp"
+
+
+namespace cv
+{
+
+namespace quality
+{
+
+/** @brief Flags to choose which algorithm MAE should use.
+ */
+enum MAEStatsFlags
+{
+    MAE_Max,
+    MAE_Mean
+};
+
+class CV_EXPORTS_W QualityMAE : public QualityBase
+{
+public:
+    /** @brief Computes MAE for reference images supplied in class constructor and provided comparison images
```

Comment:
Could you add reference to the algorithm description or paper and define what MAE means.

Reply:
We are on it!

Reply:
Please add brief algorithm description, formulae or some other reference. The solution should be obvious to other OpenCV users.

Reply:
I updated the documentation.